<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="9.485" tests="2" failures="2">
  <testsuite name="Root Suite" timestamp="2025-01-20T08:32:20" tests="0" file="cypress\e2e\network\postRequests.cy.js" time="0.000" failures="0">
  </testsuite>
  <testsuite name="Requests: POST" timestamp="2025-01-20T08:32:20" tests="2" time="9.481" failures="2">
    <testcase name="Requests: POST should be able to successfully add a new player to the backend, and then to the list of players" time="0.000" classname="should be able to successfully add a new player to the backend, and then to the list of players">
      <failure message="Timed out retrying after 4000ms: Expected to find content: &apos;Loading...&apos; within the element: &lt;div#request-status&gt; but never did." type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 4000ms: Expected to find content: 'Loading...' within the element: <div#request-status> but never did.
    at checkRequestStatus (webpack://players_crud2/./cypress/support/utils.js:56:35)
    at Context.eval (webpack://players_crud2/./cypress/e2e/network/postRequests.cy.js:25:23)]]></failure>
    </testcase>
    <testcase name="Requests: POST should not be able to add a new player if post-request fails" time="0.000" classname="should not be able to add a new player if post-request fails">
      <failure message="Timed out retrying after 4000ms: Expected to find content: &apos;Loading...&apos; within the element: &lt;div#request-status&gt; but never did." type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 4000ms: Expected to find content: 'Loading...' within the element: <div#request-status> but never did.
    at checkRequestStatus (webpack://players_crud2/./cypress/support/utils.js:56:35)
    at Context.eval (webpack://players_crud2/./cypress/e2e/network/postRequests.cy.js:63:23)]]></failure>
    </testcase>
  </testsuite>
</testsuites>